src/azul/health.py

```diff
@@ -241,3 +267,54 @@ def update_cache(self) -> None:
     @classproperty
     def all_keys(cls) -> Set[str]:
         return {p.key for p in cls.all_properties}
+
+    # noinspection PyMethodParameters
+    @classproperty
+    def receive_message_wait_time(cls) -> int:
+        """
+        The number of seconds to wait for a message to arrive in the fail queue
+        before accepting that there are zero messages to be received.
+        """
+        return 5
+
+    def archive_fail_messages(self, remaining_context_time):
+        client = boto3.client('dynamodb')
+        fail_queue = boto3.resource('sqs').get_queue_by_name(QueueName=config.fail_queue_name)
+        while remaining_context_time.get() > 0:
```

Comment:
So we still need a margin here so we don't enter another 5s request with only 1s remaining in the context.